src/sonic-config-engine/config_samples.py

```diff
@@ -50,7 +50,8 @@ def generate_t1_sample_config(data):
     data['DEVICE_METADATA']['localhost']['hostname'] = 'sonic'
     data['DEVICE_METADATA']['localhost']['type'] = 'LeafRouter'
     data['DEVICE_METADATA']['localhost']['bgp_asn'] = '65100'
-    data['LOOPBACK_INTERFACE'] = {"Loopback0|10.1.0.1/32": {}}
+    data['LOOPBACK_INTERFACE'] = {"Loopback0": {},
```

Comment:
>LOOPBACK_INTERFACE [](http://example.com/codeflow?start=10&length=18)

This line is not related to SmartSwitch. Is it a generic bugfix? If yes, suggest use a standalone PR.

Reply:
Will use seperate PR